webapp/status_server.py

```diff
@@ -96,23 +96,32 @@ def complete_task():
     'completed': True
   })
 
+
+  water = db.child('groups').child(group).child('resources').child('water').get().val()
```

Comment:
You may want to write a small function that gets you the resource values.
eg: GetResource(resourceName)
Reduces the long chains. Improves readability.